FAIMS/FAIMS3
# feat: configurable notebook name

## JIRA Ticket

[BSS-278](https://jira.csiro.au/browse/BSS-278)

## Description

Naming notebook as survey in theme.

## Proposed Changes

- Added a notebook name .env variable
- Updated all files which referenced the notebook name including URLs to reference the .env variable

## How to Test

1. Add `VITE_NOTEBOOK_NAME=survey` to `app/.env`
2. Refresh the app and verify that all references of "notebook" are now "survey"
3. (Optional) Update the variable again/remove it and refresh

## Additional Information

## Checklist
